Cargo.toml

```diff
@@ -7,10 +7,16 @@ license = "Apache-2.0"
 
 [dependencies]
 anyhow = "1.0.68"
-ethers = { version = "1.0.2", features = ["rustls"] }
-fuels = { version = "0.49.0", features = ["fuel-core-lib"] }
+async-trait = "0.1.74"
+cynic = "2.2"
+ethers = "2.0.10"
+eyre = "0.3.7"
+fuel-core-client = "0.20"
+fuels = { git = "https://github.com/VitalyVolozhinov/fuels-rs.git", branch = "master", features = ["fuel-core-lib"] }
```

Comment:
Just a reminder

Reply:
I can't they've upgraded the original with the fix but they've also added some stuff to it which breaks the querying because the bridge isn't updated.